code/obj/item/clothing/injector_mask_belt.dm

```diff
@@ -427,3 +406,153 @@ There's A LOT of duplicate code here, which isn't ideal to say the least. Should
 	check_trigger(mob/M)
 		if(isdead(M)) return 1
 		else return 0
+
+/proc/autoinjector_trigger_names(var/setType)
+	var/list/cond_types = concrete_typesof(/datum/injector_belt_condition)
+	. = new/list()
+
+	if (!setType)
+		. += "None"
+
+	for(var/A in cond_types)
+		var/datum/injector_belt_condition/C = new A()
+		. += C.name
+		if (setType)
+			.[C.name] = A
```

Comment:
you don't actually need to create the object here (just type the path), we can access compile-time properties
```suggestion
		var/datum/injector_belt_condition/C = A
		. += initial(C.name)
		if (setType)
			.[initial(C.name)] = A
```
should work